app/src/main/kotlin/com/fernandocejas/sample/features/movies/MoviesFragment.kt

```diff
@@ -31,15 +31,20 @@ import com.fernandocejas.sample.core.extension.visible
 import com.fernandocejas.sample.core.navigation.Navigator
 import com.fernandocejas.sample.core.platform.BaseFragment
 import com.fernandocejas.sample.features.movies.MovieFailure.ListNotAvailable
+import dagger.hilt.android.AndroidEntryPoint
 import kotlinx.android.synthetic.main.fragment_movies.*
 import javax.inject.Inject
 
+@AndroidEntryPoint
 class MoviesFragment : BaseFragment() {
 
-    @Inject lateinit var navigator: Navigator
-    @Inject lateinit var moviesAdapter: MoviesAdapter
+    @Inject
+    lateinit var navigator: Navigator
 
-    private val moviesViewModel: MoviesViewModel by viewModels()
+    @Inject
```

Comment:
Same here. If you feel like supporting another code style, for this PR we can keep it as it was and then we can open another one where we discuss a new format. I'm super open.

Reply:
you can see something from [Migrate to Kotlin code style](https://kotlinlang.org/docs/code-style-migration-guide.html#store-old-code-style-in-project). I change the style: 
![image](https://user-images.githubusercontent.com/21290038/116382444-7f319680-a848-11eb-8a29-b8e03fdf4ab7.png)

Reply:
if you wanna keep the old code style,  I could roll it back to style in one line. just give me a response. (*^__^*)

Reply:
I did not know about it. I would say, let's keep the old style and we merge this PR (only for consistency). And we can create another one with the new style and we do it in one shot, what do you think?

I really appreciate this contribution ❤️ 

On the other hand, once we merge this I will fix the CI.

Reply:
Yeah, you are right,  Let's keep the old code style in the PR( refactor of Hilt). one commit, one thing.   and then, you can close the PR, I'll create another PR for refactoring of Hilt and keep the old code.(^__^)

Reply:
> I did not know about it. I would say, let's keep the old style and we merge this PR (only for consistency). And we can create another one with the new style and we do it in one shot, what do you think?
> 
> I really appreciate this contribution ❤️
> 
> On the other hand, once we merge this I will fix the CI.

It's be finished that keep the old code style and upgrade the version of「Hilt」to stable.

Reply:
You are great! Thank you! ❤️